packages/block-library/src/query-loop/save.js

```diff
@@ -4,5 +4,9 @@
 import { InnerBlocks } from '@wordpress/block-editor';
 
 export default function QueryLoopSave() {
-	return <InnerBlocks.Content />;
+	return (
+		<li>
```

Comment:
This change is going to create block invalidation and we might want to introduce deprecations. It also feels a bit weird to add static HTML like that to the template. I'd have preferred if the switch to ul/li was handled on its own PR.

Reply:
I know, but since this is experimental and the block based themes are so few, do you think we need a deprecation now? Since Query now matures and evolves we might end up with many migrations really soon.

>I'd have preferred if the switch to ul/li was handled on its own PR.

Grid view cannot be possible without changing the markup.

Reply:
Can you wrap the content with `<li />` inside `render_callback` function instead?

Reply:
I will do that, yes. Thanks!